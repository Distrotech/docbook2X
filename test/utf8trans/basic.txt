– (en-dash)
— (em-dash)

“ (left double quote)
” (right double quote)

‘ (left single quote)
’ (right single quote)

… (ellipsis)

™ (trademark)
® (registered symbol)

‐ (hyphen)
