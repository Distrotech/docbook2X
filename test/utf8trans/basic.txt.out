\(en (en-dash)
\(em (em-dash)

\(lq (left double quote)
\(rq (right double quote)

\(oq (left single quote)
\(cq (right single quote)

\&... (ellipsis)

\(tm (trademark)
\(rg (registered symbol)

\(hy (hyphen)
